src/components/ActivityCard.tsx

```diff
@@ -1,45 +1,29 @@
-import * as React from 'react';
-import { styled } from '@mui/material/styles';
 import Card from '@mui/material/Card';
 import CardHeader from '@mui/material/CardHeader';
 import CardMedia from '@mui/material/CardMedia';
 import CardContent from '@mui/material/CardContent';
-import CardActions from '@mui/material/CardActions';
-import Collapse from '@mui/material/Collapse';
 import Avatar from '@mui/material/Avatar';
-import IconButton, { IconButtonProps } from '@mui/material/IconButton';
 import Typography from '@mui/material/Typography';
-import FavoriteIcon from '@mui/icons-material/Favorite';
-import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import IconButton from '@mui/material/IconButton';
 import MoreVertIcon from '@mui/icons-material/MoreVert';
-import defaultimg from './default.png'
-import { Button } from '@mui/material';
 import PersonAddIcon from '@mui/icons-material/PersonAdd';
+import PeopleAltIcon from '@mui/icons-material/PeopleAlt';
+import PlaceIcon from '@mui/icons-material/Place';
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import { Button, Divider } from '@mui/material';
+import { ActivityItem } from '../api/activities.api.types';
+import { Box } from '@mui/system';
 
-interface ExpandMoreProps extends IconButtonProps {
-  expand: boolean;
-}
-
-const ExpandMore = styled((props: ExpandMoreProps) => {
-  const { expand, ...other } = props;
-  return <IconButton {...other} />;
-})(({ theme, expand }) => ({
-  transform: !expand ? 'rotate(0deg)' : 'rotate(180deg)',
-  marginLeft: 'auto',
-  transition: theme.transitions.create('transform', {
-    duration: theme.transitions.duration.shortest,
-  }),
-}));
 
-export default function ActivityCard() {
-  const [expanded, setExpanded] = React.useState(false);
+interface ActivityCardProps {
+  activityItem: ActivityItem;
+}
 
-  const handleExpandClick = () => {
-    setExpanded(!expanded);
-  };
+export default function ActivityCard(props: ActivityCardProps) {
+  const { activityItem } = props;
 
   return (
-    <Card sx={{ mb: '30px'}}>
+    <Card sx={{ width: '900px', ml: 'auto', mr: 'auto', mb: '30px'}}>
```

Comment:
It was quite some time when I last used react mui: is it possible to extract these styles somehow? 🤔